src/command_modules/azure-cli-storage/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+2.4.3
++++++
+* `storage container generate-sas`: Fix missing account key
```

Comment:
Does this actually fix the problem or just improve the error message?

Reply:
The root cause of the problem is that generate_container_shared_access_signature need account key. So auth_mode login won't work as it only provide the token. So I removed the auth_mode of login. The user has to use the account key to generate sas.

Reply:
So they will see a better error message?

Reply:
The user has to use account key to call generate-sas. No auth_mode login is allowed and no error will show any more.